credentialbuilder/src/main/java/org/apache/ranger/credentialapi/CredentialReader.java

```diff
@@ -17,80 +17,82 @@
  * under the License.
  */
 
- package org.apache.ranger.credentialapi;
-import java.util.ArrayList;
-import java.util.List;
+package org.apache.ranger.credentialapi;
 
 import org.apache.commons.lang3.StringUtils;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.security.alias.CredentialProvider;
 import org.apache.hadoop.security.alias.CredentialProviderFactory;
 import org.apache.hadoop.security.alias.JavaKeyStoreProvider;
 
-public class CredentialReader {
+import java.util.ArrayList;
+import java.util.List;
 
-  public static String getDecryptedString(String CrendentialProviderPath,String alias, String storeType) {
-	  String credential=null;
-	  try{
-		  if(CrendentialProviderPath==null || alias==null){
-			  return null;
-		  }		  		
-		  char[] pass = null;
-		  Configuration conf = new Configuration();
-		  String crendentialProviderPrefixJceks=JavaKeyStoreProvider.SCHEME_NAME + "://file";
-		  String crendentialProviderPrefixLocalJceks="localjceks://file";
-		  crendentialProviderPrefixJceks=crendentialProviderPrefixJceks.toLowerCase();
+public class CredentialReader {
+    private CredentialReader() {
+    }
 
-		  String crendentialProviderPrefixBcfks = "bcfks" + "://file";
-		  String crendentialProviderPrefixLocalBcfks= "localbcfks" + "://file";
-		  crendentialProviderPrefixBcfks=crendentialProviderPrefixBcfks.toLowerCase();
-		  crendentialProviderPrefixLocalBcfks=crendentialProviderPrefixLocalBcfks.toLowerCase();
+    public static String getDecryptedString(String credentialProviderPath, String alias, String storeType) {
+        String credential = null;
+        try {
+            if (credentialProviderPath == null || alias == null) {
+                return null;
+            }
+            char[] pass = null;
```

Comment:
@pradeepagrawal8184  - did you reformat the code using codestyle template at dev-support/RangerCodeScheme-IntelliJ.xml? It is important to maintain consistency across the code base.